consensus/istanbul/validator/weighted.go

```diff
@@ -280,7 +281,12 @@ func (valSet *weightedCouncil) SubGroupSize() uint64 {
 	return valSet.subSize
 }
 
+// SetSubGroupSize sets committee size of the valSet.
 func (valSet *weightedCouncil) SetSubGroupSize(size uint64) {
+	if size == 0 {
+		logger.Error("cannot assign committee size to 0")
```

Comment:
Crit? 
```suggestion
		logger.Crit("cannot assign committee size to 0")
```

Reply:
I think `error` is enough. It doesn't change `valSet.subSize` value when `size` is 0.